tests/checker/__init__.py

```diff
@@ -90,42 +99,44 @@ def log_url(self, url_data):
         """
         Append logger output to self.result.
         """
-        if self.has_part('url'):
+        if self.has_part("url"):
             url = "url %s" % url_data.base_url
             self.result.append(url)
-        if self.has_part('cachekey'):
+        if self.has_part("cachekey"):
             cache_key = url_data.cache_url if url_data.cache_url else None
             self.result.append("cache key %s" % cache_key)
-        if self.has_part('realurl'):
+        if self.has_part("realurl"):
             self.result.append("real url %s" % url_data.url)
-        if self.has_part('name') and url_data.name:
+        if self.has_part("name") and url_data.name:
             self.result.append("name %s" % url_data.name)
-        if self.has_part('base') and url_data.base_ref:
+        if self.has_part("base") and url_data.base_ref:
             self.result.append("baseurl %s" % url_data.base_ref)
-        if self.has_part('info'):
+        if self.has_part("info"):
```

Comment:
I hate the noise Black creates by changing the quote characters on all the strings.  And they make the diff so huge.  I'm sure I'll give up somewhere in the middle.

Reply:
i also find this change annoying, because i usually type single quotes because they are faster (one keystroke instead of shift+keystroke) on my keyboard.

the black docs have a section on this:

https://black.readthedocs.io/en/stable/the_black_code_style.html#strings

their argument ias that double-quotes are less ambiguous and that you can always type single quotes and let black fix the formatting if you don't like to type double-quotes, which is reasonable.

That said, there's an escape hatch in black for this:

> If you are adopting Black in a large project with pre-existing string conventions (like the popular “[single quotes for data, double quotes for human-readable strings](https://stackoverflow.com/a/56190)”), you can pass `--skip-string-normalization` on the command line. This is meant as an adoption helper, avoid using this for new projects.

Do we want to do that?

Reply:
Yes please, anything for a smaller diff.

(I won't mind if there's a subsequent PR thst does only quote changed, separately, if anyone wants to do that.  Personally, I'm not yet convinced by Black's arguments about quotes.)